ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -60,7 +70,23 @@ data = yaml.load(open(path));
 data["compute"] = [ { "name": "worker", "replicas": 0 } ];
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
+### Update base domain setting
+if [[ -v IS_QE_XPN ]]; then
```

Comment:
I think only when interact with two base domains, need to update install-config.yaml, I guess the dev's original logic is using one base domain even for xpn install,  that 's why I never found a place how dev configure the 2nd base domain. So this condition maybe need a bit update, I have some idea about how to handle base domain, we can discuss more on slack.